config/locales/en.yml

```diff
@@ -79,6 +79,8 @@ en:
         unauthorized_client: 'The client is not authorized to perform this request using this method.'
         access_denied: 'The resource owner or authorization server denied the request.'
         invalid_scope: 'The requested scope is invalid, unknown, or malformed.'
+        invalid_code_challenge_method: 'The transformation method must be plain or S256.'
```

Comment:
Just a nit pick for consistency, but `The code challenge method must be "plain" or "S256"`